content/agent/kubernetes/host_setup.md

```diff
@@ -0,0 +1,48 @@
+## Setup Kubernetes State
```

Comment:
This file seems inappropriately named (both file name and in the navigation) as "Host Setup"